tests/ui/test_ui_tools.py

```diff
@@ -1244,6 +1244,7 @@ def test_text_content(self, mocker,
                 count_str)
         assert len(text[0]) == len(expected_text) == (width - 1)
         assert text[0] == expected_text
+        assert top_button._w._original_widget._wrap_mode == 'any'
```

Comment:
Primarily, to keep computation of showing unread count at the right position simple. Otherwise, we will have to take wrapping of words into account which is currently not included in the code.

Reply:
I can see that this could confuse the placement of the unread counts. That said, the styling of the words looks pretty bad if the text wraps mid-word.

I would put this 'why' into the commit title/summary in any case.

Reply:
Of course! done :+1: